unit-tests/unit-tests-post-processing-from-bag.cpp

```diff
@@ -541,19 +541,19 @@ TEST_CASE("Record point cloud software-device all resolutions", "[record-bag][po
     record_frames_all_res(record_block, "[pointcloud]_all_combinations_depth_color.bag");
 }
 
-TEST_CASE("Test align color to depth from recording", "[software-device][align]")
+TEST_CASE("Test align color to depth from recording", "[software-device-disabled][align]")
```

Comment:
We're not removing? Just marking so won't run?

Reply:
We have 1/2 more legacy tests left to convert.
So we decided to disable those we don't run until we will remove all of this mechanism..